SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -61,18 +61,14 @@ class FLocalDeploymentManager
 
 	bool bLocalDeploymentRunning;
 	bool bSpatialServiceRunning;
+	bool bSpatialServiceInProjectDirectory;
 
 	bool bStartingDeployment;
 	bool bStoppingDeployment;
 
 	bool bStartingSpatialService;
 	bool bStoppingSpatialService;
 
-	FDateTime LastSpatialServiceCheck;
```

Comment:
I assume these variables are unrelated to PR but here only as a part of cleanup?

Reply:
Yeah this was just some cleanup of leftover unused vars